tsdb/batcher.go

```diff
@@ -23,8 +28,9 @@ func NewPointBatcher(sz int, d time.Duration) *PointBatcher {
 	return &PointBatcher{
 		size:     sz,
 		duration: d,
-		in:       make(chan Point),
-		out:      make(chan []Point),
+		stop:     make(chan struct{}),
+		in:       make(chan Point, 1000000),
```

Comment:
Why are you doing this? By doing this you will allow points pile up in the batcher even if the downstream component cannot drain them. 

This channel is unbuffered by design.